packages/identifier-resolution/src/agent/IdentifierResolution.ts

```diff
@@ -1,5 +1,5 @@
 import { IAgentContext, IAgentPlugin, IDIDManager, IKeyManager } from '@veramo/core'
-import { schema } from '..'
+import { ExternalIdentifierOIDFEntityIdOpts, ExternalIdentifierOIDFEntityIdResult, schema } from '..'
```

Comment:
small note, these should be moved the the types import

Reply:
fixed, schema is not available in types though